python/llm/src/bigdl/llm/utils/ipex_importer.py

```diff
@@ -35,10 +35,19 @@ def is_xpu_version_installed():
         """
         # Check if xpu version installed
         try:
+            # Check if bigdl-core-xe is installed
             distribution('bigdl-core-xe')
             return True
         except PackageNotFoundError:
-            return False
+            # bigdl-core-xe not found
+            # Check if bigdl-core-xe-21 is installed
+            try:
+                distribution('bigdl-core-xe-21')
+                return True
```

Comment:
Can we use `importlib.util.find_spec` instead?

Reply:
`importlib.util.find_spec` is not useful here.
I tested it on arc14 with the following code
```
        if importlib.util.find_spec('bigdl-core-xe') is not None:
            return True
        else:
            print("intel_extension_for_pytorch not found")
            return False
```
`importlib.util.find_spec` doesn't find ipex
```
(ziteng-test) arda@arda-arc14:~/anaconda3/envs/ziteng-test/lib/python3.9/site-packages/bigdl/llm/utils$ python ipex_importer.py
intel_extension_for_pytorch not found
```